bin/main.rs

```diff
@@ -77,6 +84,12 @@ struct Args {
     /// The NATS JetStream domain to connect to
     #[arg(short = 'd', env = "WADM_JETSTREAM_DOMAIN")]
     domain: Option<String>,
+
+    #[arg(long = "storage-nkv-url", env = "STORAGE_NATSKV_URL")]
+    storage_nkv_url: Option<String>,
+
+    #[arg(long = "storage-nkv-prefix", env = "STORAGE_NATSKV_PREFIX")]
```

Comment:
we should probably add some help text to this. Not everyone will know what this prefix is or how it impacts the deployment.

Reply:
oversight on my part, will add some help text

- [ ] Add help text for `--storage-*` options